keras/src/backend/openvino/excluded_concrete_tests.txt

```diff
@@ -95,6 +96,7 @@ NumpyOneInputOpsCorrectnessTest::test_argmin
 NumpyOneInputOpsCorrectnessTest::test_argpartition
 NumpyOneInputOpsCorrectnessTest::test_argsort
 NumpyOneInputOpsCorrectnessTest::test_array
+NumpyOneInputOpsCorrectnessTest::test_average
 NumpyOneInputOpsCorrectnessTest::test_bincount
```

Comment:
Here you should only remove lines corresponding to bincount tests.
Now you are adding for `average` for some reason

Reply:
sorry, while resolving the merge conflict I took a merge with the master branch and it was an incoming change so I did kept a combination of both current and incoming, I will remove it. 
Thank you

Reply:
fixed